### DISCUSSION TITLE
How can I enable HTTPS for my www subdomain alongside the root?

### DISCUSSION THREAD

**Discussion by JoshuaFox** [community/community#23944] [Other Feature Feedback, Questions, & Ideas]
<p>Github <a href="https://github.blog/changelog/2021-04-08-github-pages-can-now-automatically-secure-the-www-variant-of-your-custom-domain/" rel="noopener nofollow ugc">now can automatically support</a> HTTPS for the www subdomain alongside the root domain.</p>
<p><a href="https://www.joshuafox.com" rel="noopener nofollow ugc">https://www.joshuafox.com</a>  shows  an SSL certificate error, while the root domain is fine.</p>
<p>The DNS records seem correct, as shown by <code>dig</code>.</p>
<p>Setting for Google Pages is as below.</p>
<p>The www subdomain is not  mentioned there, and I think that it is supported (without HTTPS) simply with a  CNAME DNS zone.</p>
<p>How do I enable HTTPS support for my www subdomain as well as for the root domain?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/2/d2d3d986e26bc3a19308a1c0d65db41f1a5aa647.png" data-download-href="/uploads/short-url/u549AIIk3gCcSoj8EntRUTQvMYn.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/4542591/181080342-9821218a-9361-45f3-987b-7164a9dde3da.png" alt="image" data-base62-sha1="u549AIIk3gCcSoj8EntRUTQvMYn" width="690" height="264" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/2/d2d3d986e26bc3a19308a1c0d65db41f1a5aa647_2_690x264.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/2/d2d3d986e26bc3a19308a1c0d65db41f1a5aa647_2_1035x396.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/2/d2d3d986e26bc3a19308a1c0d65db41f1a5aa647_2_1380x528.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/2/d2d3d986e26bc3a19308a1c0d65db41f1a5aa647_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2054×786 85.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by otter-computer** [comment, marked as answer]
<p>It looks like your <a href="https://www.whatsmydns.net/#CNAME/www.joshuafox.com">www domain’s CNAME is pointing to your apex domain</a>. You’ll need to point this towards your GitHub domain <code>JoshuaFox.github.io</code> instead. Once you’ve changed that, and waited for it to propagate, you’ll then need to visit your repository’s settings page for Pages. Visiting that page should be enough to trigger GitHub to re-request an SSL certificate.</p>

**Comment by JoshuaFox** [comment]
<p>Thank you! That did it! <a href="https://www.joshuafox.com" rel="noopener nofollow ugc">https://www.joshuafox.com</a> and  <a href="https://joshuafox.com" rel="noopener nofollow ugc">https://joshuafox.com</a> both work</p>
